### PR TITLE
Create to-do app

### PR DESCRIPTION
**Pull request by Yosefgeda**
- Project setup with webpack.
- Displays to-do lists in an array.

### CODE REVIEW

File: src/index.js [hunk 1]
```diff
@@ -0,0 +1,34 @@
...
+  {
+    description: 'Pay Bills',
+    completed: false,
+    index: 3,
+  },
+];
+const generatedList = document.querySelector('.generated-list');
+
+const displayTodo = () => {
+  const displayTodo = list.map((item) => `
```
Comment (gemmen29) [inline src/index.js]: - [ ] Kindly sort the list before displaying it as per this [requirement](https://github.com/microverseinc/curriculum-javascript/blob/main/todo-list/m1_list_structure.md#:~:text=The%20list%20should%20appear%20in%20order%20of%20the%20index%20values%20for%20each%20task.). The list should appear in order of the index values for each task.

File: src/index.html [hunk 2]
```diff
@@ -0,0 +1,24 @@
...
+                    <input type="text" placeholder="Add your list...">
+                    <button class="submit-btn">Add</button>
+                </div>
+        </div>
+
+        <div class="generated-list">
+
+        </div>
+        
+    </div>
```
Comment (gemmen29) [inline src/index.html]: - [ ] Kindly try to make your design a clone from the guidelines video as per this [requirement](https://github.com/microverseinc/curriculum-javascript/blob/main/todo-list/m1_list_structure.md#:~:text=Your%20list%20should%20be%20a%20clone%20of%20the%20part%20of%20the%20minimalist%20project%20captured%20in%20the%20video%20below.).

**Review by gemmen29 — CHANGES_REQUESTED** [review]
Hi @Yosefgeda, 

Good job so far!
There are some issues that you still need to work on to go to the next project but you are almost there!

## Highlights 💯 
- No linters errors ✔️ 
- Descriptive pull request ✔️ 
- Descriptive commit messages ✔️ 

## Required Changes ♻️
_Check the comments under the review._



## Optional suggestions
_Every comment with the [OPTIONAL] prefix is not crucial enough to stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better._

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Review by Favourezeugwa — APPROVED** [review]
## STATUS: APPROVED 🥇 🏆 🍹 

Hi @Yosefgeda ,

I am Favour 👩🏽 , your reviewer for this PR.

Your project is complete!  There is nothing else to say other than... it's time to merge it :shipit: 
Yayyy! 💃🏽, Congratulations! 🎉 🎉🎉

happy

## Highlights 💯 
- No Linter errors ✔️
- Good commit messages ✔️
- Professional Readme file ✔️
- Descriptive PR ✔️ 

## Optional suggestions

_Every comment with the [OPTIONAL] prefix won't stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better. Some of them were simply missed by the previous reviewer and addressing them will really improve your application._

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Pull request merged**